package.json

```diff
@@ -16,6 +16,7 @@
     "lint": "yarn lint-sol & yarn lint-ts --no-error-on-unmatched-pattern",
     "clean": "hardhat clean",
     "build": "yarn run clean && yarn run compile && tsc",
+    "reinstall": "yarn cache clean && rm -rf node_modules && yarn",

```

Comment:
this is not added here. this is a locally run script and has nothing to do with this repo